src/measurement/store.ts

```diff
@@ -145,6 +136,7 @@ export class MeasurementStore {
 
 		await Promise.all([
 			this.redis.hDel('gp:in-progress', ids),
+			ids.map(id => this.redis.del((getMeasurementKey(id, 'probes_awaiting')))),
```

Comment:
This should be `...` like the line below, no?

Reply:
Indeed, fixed.